### DISCUSSION TITLE
sensor.openwb_uptime is spamming the database

### DISCUSSION THREAD

**Discussion by northalpha** [ChristophCaina/HA-OpenWB-MQTT#37] [Q&A] [answered]
Hi first of all, aweseome addon/integration!

using the latest update the sensor `sensor.openwb_uptime` is updated minutely, which by itself is no issue at all, but maybe the update frequency is to often. This is spamming/fillup the history/database from HA with useless information.

may i ask to decrease the frequency to maybe daily? 

I am very aware of that i might just deactivate the sensor to get rid f these messages, but just as an idea for not that "experienced" users "just using your integration" :)

**Comment by ChristophCaina** [comment, marked as answer]
Hi @northalpha,

first of all, the credits should go to @ a529987659852 who has created the addon.
I've just forked it trying to create some fixes and additional sensors :)

I had another thought of your reply regarding the Uptime Sensor... and I think, it could work if we MAYBE remove the minutes from the sensor - in that case, the integration would still receive the MQTT Topic, but the update frequency of the sensor should change to hourly.

On the other hand, monitoring is what it is... and if we want to monitor the uptime of the Wallbox, then it SHOULD include the whole uptime information (in my opinion)
I have several other sensors where I receive uptime information every minute... and yes, I can see that this is causing a lot of data in the HA Database...

as an Workaround, I would suggest to exclude the openwb uptime sensor from the HomeAssistant Recorder:

```
recorder:
  exclude:
    entity_globs:
      - sensor.*uptime*
```
or:

```
recorder:
  exclude:
    entities:
      - sensor.openwb_uptime
 ```
 
![grafik](https://user-images.githubusercontent.com/26391061/195195491-ae1eec9b-799d-4c73-b1cc-e7606ca29b03.png)

**Reply by northalpha** [reply]
I agree with your proposal to "just exclude" this from the recorder/database after a good sleep und morning coffee. messing around with metrics from the source to the user should not be done, if we/you "just" use the values coming on on mqtt beeing published by openwb.

maybe you could include this as an informational in the next release or something like this to explain what this might be doing to an SD Card on a raspberry pi HA instance. 

i will go with the recorder exclude entity option, thanks for taking the time! :)
